src/stores/registerChallenge.ts

```diff
@@ -0,0 +1,48 @@
+import { defineStore } from 'pinia';
+
+// types
+import { FormPersonalDetailsFields } from '../components/types/Form';
+
+const emptyFormPersonalDetails: FormPersonalDetailsFields = {
+  firstName: '',
```

Comment:
Instead empty string use `null` please.

Reply:
Updated. This approach, however, bloats the code a bit, since we need to provide a getter for each of these propeties, which ensures, that the values can get to the input as a `string` type, and not as a `string | null` type.
This is required when using v-model on Quasar inputs.

Reply:
Store should work independently from Quasar components.

It is important have same type of init store key value across code. We use in "previous" PR for store init key value `null`.

If you want use empty string aka `''`, it is important make this changes global and for future code too.

Reply:
I agree with both your statements.
Does it mean that you prefer to init all store keys with `null` regardless of type?
Is there an advantage of doing this vs. for example:
* empty string for strings
* `0` for numbers
* `[]` for arrays
* `null` for objects and booleans?
In my opinion, this simplifies the type inference, but I am also ok with using `null` (which is the current solution).

Reply:
I agree, you can use empty string `''` as init value. 

object : {} 

null for boolean is interpreted as false

Reply:
For boolean type (checkbox input), we can use init value according context. If some checkbox widget is by default unchecked, store key value would be `false`and vice-versa.